client/js/game.js

```diff
@@ -23647,8 +23648,9 @@ function Game() {
         xmlHTTP.send();
     }
     var bsynking=false;
+    var bsyncTime = Date.now() - 5*60*1000;
     this.bsync = function () {
-        if (bsynking) return;
+        if (bsynking || bsyncTime + 5*6*1000 > Date.now()) return;
```

Comment:
You really mean 30 seconds ? Or that's a typo ?

Reply:
^^U